css/font.css

```diff
@@ -0,0 +1,14 @@
+﻿@font-face {
+  font-family: 'GothaPro';
+  src: url('../fonts/GothaProReg.eot');
+  src: url('../fonts/GothaProReg.woff') format('woff'), url('../fonts/GothaProReg.woff2') format('woff2'),url('../fonts/GothaProReg.ttf') format('truetype'), url('../fonts/GothaProReg.svg') format('svg');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected single space after "," in a single-line list (value-list-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/axion_ann/pullRequest?prid=5594413)